package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "music-api",
+  "version": "1.0.0",
+  "description": "Build front end for Music-Api site. This music Api provides music albums for different genres.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./configs/sequelize.js",
+    "lint": "./node_modules/.bin/eslint --format codeframe .",
+    "lint:fix": "./node_modules/.bin/eslint --fix --format codeframe .",
```

Comment:
Your eslint command doesn't work since you didn't add a config file

![image](https://user-images.githubusercontent.com/5775716/84066360-c8a89700-a993-11ea-8450-6c59bbfc9b1b.png)

Had you configured it properly it would have found your 92 issues for you all of which can be fixed with the `lint:fix` command.

![image](https://user-images.githubusercontent.com/5775716/84066487-02799d80-a994-11ea-80fb-ba53d748c275.png)